### PR TITLE
Improve extensions part in trainer tutorial

### PR DESCRIPTION
**Pull request by niboshi**
Fixed typos, markups, and improved some wording and appearance.

### CODE REVIEW

File: docs/source/tutorial/trainer.rst [hunk 1]
```diff
@@ -197,45 +199,42 @@ To use these wide variety of tools for your training task, pass :class:`~chainer
...
+     This extension outputs the spcified values to the standard output.
 
-:class:`~chainer.training.extensions.PlotReport` plots the values specified by its arguments saves it as a image file which has the same naem as the :attr:`~chainer.training.extensions.PlotReport.file_name` argument.
+  :class:`~chainer.training.extensions.PlotReport`
+     This extension plots the values specified by its arguments and saves it as a image file.
 
-----
 
-Each :class:`~chainer.training.Extension` class has different options and some extensions are not mentioned here. And one of other important feature is, for instance, by using the :attr:`~chainer.training.Extension.trigger` option, you can set individual timings to fire the :class:`~chainer.training.Extension`. To know more details of all extensions, please take a look at the official document: `Trainer extensions <reference/extensions.html>_`.
+This is not the exhaustive list of built-in extensions.
```
Comment (bkvogel) [inline docs/source/tutorial/trainer.rst]: This is not the exhaustive list -> This is not an exhaustive list

**Comment by bkvogel** [comment]
Thanks for the PR and sorry for the delay in reviewing. Please resolve the conflicts and I will continue with the review.

**Comment by niboshi** [comment]
Conflict resolved.

File: docs/source/tutorial/trainer.rst (hunk 1 above)
Comment (niboshi) [reply docs/source/tutorial/trainer.rst]: Fixed!

**Comment by bkvogel** [comment]
Thanks. LGTM!

**Pull request merged**